meta/main.yml

```diff
@@ -1,68 +1,25 @@
 # SPDX-License-Identifier: MIT
 ---
 galaxy_info:
-  # Replace with role's author name:
-  author: John Doe <jdoe@corp.com>
-  # Replace with the real description of what is role's purpose:
-  description: Basic template for Linux system roles
-  # Replace with the company the role's author is member of:
-  company: John Doe, Inc.
-
-  # If the issue tracker for your role is not on github, uncomment the next
-  # line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
+  author: Sergio Correia <scorreia@redhat.com>
+  description: >
+    Ansible role for configuring attestation servers
```

Comment:
Do you plan to make the role work for other than keylime attestation servers? We must configure a different naming in this case, because the current name implies only keylime server support.

Reply:
I fixed the description. This role is intended to deploy Keylime attestation servers; that word was missing there.